apps/bmd/src/types/kiosk-browser.d.ts

```diff
@@ -1,3 +1,12 @@
+// Disable `no-var` because using `var` ensures `kiosk` is a property on
+// `globalThis`, which makes it available both as plain `kiosk` and as
+// `window.kiosk`. Using `const` or `let`, as eslint suggests, will not make it
+// available at all. An alternative would be to add `kiosk` as a property to
+// the `Window` interface, but then we couldn't refer to `kiosk` without
+// `window`.
```

Comment:
I thought we had decided to always prefix with `window` so as not to confuse with any local variable… but I don't feel strongly about this. If we don't prefix with `window`, what is a good way for a developer — fresh to the code — to be able to know where `kiosk` is coming from?

Reply:
Well, if it's not declared in the scope/file/module then it's gotta be global. But this isn't new. I just moved this chunk from the bottom of the file.k